pyopenms_viz/testing/BokehSnapshotExtension.py

```
@@ -136,38 +139,14 @@ def compare_json(json1, json2):
-    ) -> None:
-        # see https://github.com/tophat/syrupy/blob/f4bc8453466af2cfa75cdda1d50d67bc8c4396c3/src/syrupy/extensions/base.py#L161
-
-        filepath, data = (
-            snapshot_collection.location,
-            next(iter(snapshot_collection)).data,
-        )
-        with open(filepath, "w") as f:
-            f.write(data)
-
-    def serialize(self, data: SerializableData, **kwargs: Any) -> str:
+    def serialize(self, data: SerializableData, **kwargs: Any) -> bytes:
```

Ruff (0.14.11)

Unused method argument: `kwargs`

(ARG002)